src/main/java/io/pravega/connectors/flink/FlinkPravegaReader.java

```diff
@@ -642,11 +661,10 @@ protected EventStreamClientFactory createEventStreamClientFactory() {
      * @param readerId the readerID to use.
      * @return An instance of {@link EventStreamReader}
```

Comment:
Add some more javadoc to explain it is an `eventStreamReader` reading raw bytes now.